.goreleaser.yml

```diff
@@ -8,6 +8,7 @@ archives:
         format: zip
 builds:
   - id: webhook-go
+    ldflags: -s -w -X 'github.com/voxpupuli/webhook-go/cmd.version={{.Version}}'
```

Comment:
mhm what does this do?

Reply:
That sets the `version` variable in the `cmd/root.go` file to the value of the release when the binary is compiled. That way, the `--version` option will return the actual release version